default.nix

```diff
@@ -1,32 +1,7 @@
-let
-  hostPkgs = import <nixpkgs> {};
-  pinnedPkgs = hostPkgs.fetchFromGitHub {
-    owner = "NixOS";
-    repo = "nixpkgs-channels";
-    rev = "ee28e35ba37ab285fc29e4a09f26235ffe4123e2";
-    sha256 = "0a6xrqjj2ihkz1bizhy5r843n38xgimzw5s2mfc42kk2rgc95gw5";
-  };
-
-in { nixpkgs ? import pinnedPkgs {}
-   , compiler ? "ghc822"
-   , doProfiling ? false
-   , doBenchmark ? false }:
-
-let
-
-  inherit (nixpkgs) pkgs;
-
-  haskellPackages = let hpkgs = pkgs.haskell.packages.${compiler}; in
-    hpkgs // {
-      mkDerivation = args: hpkgs.mkDerivation (args // {
-        enableLibraryProfiling = doProfiling;
-        enableExecutableProfiling = doProfiling;
-      });
-    };
-
-  pkg = haskellPackages.developPackage {
+{ pkgs ? import <nixpkgs> {} }: pkgs.haskellPackages.developPackage {
```

Comment:
This change loses the pinning, and the ability to enable profiling. Do you overwrite default.nix with something you intended only to use locally?